extensions/wikia/VideoHandlers/apiwrappers/FiveminApiWrapper.class.php

```diff
@@ -3,7 +3,7 @@
 class FiveminApiWrapper extends ApiWrapper {
 
 	protected static $RESPONSE_FORMAT = self::RESPONSE_FORMAT_XML;
-	protected static $API_URL = 'http://api.5min.com/video/$1/info.xml?thumbnail_sizes=true';
+	protected static $API_URL = 'https://api.5min.com/video/$1/info.xml?thumbnail_sizes=true';
 	protected static $CACHE_KEY = 'fiveminapi';
 	protected static $aspectRatio = 1.3636;	// 480 x 352
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app